resources/views/formfields/rich_text_box.blade.php

```diff
@@ -1,4 +1,4 @@
-<textarea @if($row->required == 1) required @endif class="form-control richTextBox" data-name="{{ $row->display_name }}"  name="{{ $row->field }}" id="richtext{{ $row->field }}">
+<textarea @if($row->required == 1) @endif class="form-control richTextBox" data-name="{{ $row->display_name }}"  name="{{ $row->field }}" id="richtext{{ $row->field }}">
```

Comment:
If you're going to remove `required`, you need to remove the empty `@if() ... @endif`` around it.

Reply:
ouhc, in a hurry missed